{"name": "Тест на получение заказов конкретного пользователя без авторизации", "status": "passed", "start": 1715686144964, "stop": 1715686145543, "uuid": "af4e42e4-7c71-4a34-90d2-3a92ed2f92fe", "historyId": "df8ddbe45775d5a4082a1955c2d3012a", "testCaseId": "df8ddbe45775d5a4082a1955c2d3012a", "fullName": "tests.test_get_orders_concrete_user.TestGetOrdersConcreteUser#test_get_orders_concrete_user_without_auth_return_error_message_and_401", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_get_orders_concrete_user"}, {"name": "subSuite", "value": "TestGetOrdersConcreteUser"}, {"name": "host", "value": "MacBook-Pro.local"}, {"name": "thread", "value": "19703-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_get_orders_concrete_user"}]}